extensions/arc/deployment/src/test/java/io/quarkus/arc/test/deployment/TestInvalidPostConstruct.java

```diff
@@ -0,0 +1,38 @@
+package io.quarkus.arc.test.deployment;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.spi.DefinitionException;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.arc.Unremovable;
+import io.quarkus.test.QuarkusUnitTest;
+import io.smallrye.mutiny.Uni;
+
+public class TestInvalidPostConstruct {
```

Comment:
These tests are ok but relatively "expensive" in terms of added value and time needed to run them. It might be better to add a test directly in the [ArC test module](https://github.com/quarkusio/quarkus/tree/main/independent-projects/arc/tests). Something like https://github.com/quarkusio/quarkus/blob/main/independent-projects/arc/tests/src/test/java/io/quarkus/arc/test/validation/InitializerMethodMarkedProducerTest.java

Reply:
Let me try to move them.